lib/utils/testem-events.js

```diff
@@ -152,31 +161,56 @@ class TestemEvents {
     };
   }
 
+  /**
+   * Creates a file with targetJsonObject
+   *
+   * @param {*} fileName
+   * @param {*} targetJsonObject
+   * @param {*} option
+   */
+  _writeJsonToFile(fileName, targetJsonObject, option = {}) {
+    try {
+      fs.writeJsonSync(fileName, targetJsonObject, option);
+    } catch (err) {
+      err.file =  err.file || fileName;
+      throw err;
+    }
+  }
+
+  /**
+   * Returns a sorted module metadata from statemanager.
+   */
+  _getSortedModuleMetaData(moduleMetadata) {
```

Comment:
nit:  this appears to be a helper method, not reliant on `this` nor to be overridden by sub-classes. We should extract this from the prototype, and instead make it a module-level "pure" function.